AUTHORS

```diff
@@ -1,39 +1,61 @@
-Akshay Venkatesh
-Alexander Margolin <alex.margolin@huawei.com>
-Alexander Mikheev <alexm@mellanox.com>
+Akshay Venkatesh <akvenkatesh@nvidia.com>
+Alex Margolin <alex.margolin@huawei.com>
+Alex Mikheev <alexm@mellanox.com>
 Alina Sklarevich <alinas@mellanox.com>
-Andrey Maslennikov
-Artem Polyakov
-Artemy Kovalyov
+Andrey Maslennikov <andreyma@mellanox.com>
+Artem Polyakov <artemp@mellanox.com>
+Artemy Kovalyov <artemyko@mellanox.com>
 Aurelien Bouteiller <bouteill@icl.utk.edu>
-Devendar Bureddy
+Brad Benton <bradford.benton@gmail.com>
+Corey J. Nolet <cjnolet@gmail.com>
+Devendar Bureddy <devendar@mellanox.com>
+Dmitry Gladkov <dmitrygla@mellanox.com>
+Doug Jacobsen <dmjacobsen@lbl.gov>
 Elad Persiko <eladpe@mellanox.com>
 Eugene Voronov <eugene@mellanox.com>
 Evgeny Leksikov <evgenylek@mellanox.com>
-Gilles Gouaillardet
+Gilbert Lee <gilbert.lee@amd.com>
+Gilles Gouaillardet <gilles.gouaillardet@gmail.com>
 Graham Lopez <lopezmg@ornl.gov>
 Guy Shattah <sguy@mellanox.com>
 Hiroyuki Sato <hiroysato@gmail.com>
 Howard Pritchard <howardp@lanl.gov>
-Igor Ivanov <igori@mellanox.com>
-Ilya Nelkenbaum
-Jeff Daily
-John Snyder <jsnyder@cs.duke.edu>
-Khaled Hamidouche
-Manjunath Gorentla Venkata <manjugv@ornl.gov>
+Igor Ivanov <Igor.Ivanov@itseez.com>
+Ilya Nelkenbaum <ilyan@mellanox.com>
+Jason Gunthorpe <jgg@mellanox.com>
+Jeff Daily <jeff.daily@amd.com>
+John Snyder <jms285@duke.edu>
+Keisuke Fukuda <kfukuda@preferred.jp>
+Ken Raffenetti <raffenet@mcs.anl.gov>
+Khaled Hamidouche <Khaled.Hamidouche@amd.com>
+Konstantin Belousov <kib@freebsd.org>
+Luis E. P <l31g@hotmail.com>
+Manjunath Gorentla Venkata <manjugv@gmail.com>
+Marek Schimara <Marek.Schimara@bull.net>
 Matthew Baker <bakermb@ornl.gov>
 Mike Dubman <miked@mellanox.com>
-Mikhail Brinskii <mikhailb@mellanox.com>
+Mikhail Brinskiy <mikhailb@mellanox.com>
 Nathan Hjelm <hjelmn@lanl.gov>
 Netanel Yosephian <netanelyo@mellanox.com>
-Pavel Shamis <pavel.shamis@arm.com>
+Pak Lui <5041261+paklui@users.noreply.github.com>
```

Comment:
this email is weird, should be `<Pak.Lui@amd.com>`
@paklui

Reply:
@paklui - is this ok to replace GitHub blob with you actual email ? @yosefe I cannot expose developers email beyond what is officially registered at GitHub. So we have two option here: get ok from Pak, or remove it all together

Reply:
Yes I am ok with that. Thanks.